public/index.html

```diff
@@ -0,0 +1,370 @@
+                <ul class="social-media">
+                    <li><a href=""><span class="socicon">b</span></a></li>
+                    <li><a href=""><span class="socicon">a</span></a></li>
+                    <li><a href=""><span class="socicon">D</span></a></li>
+                    <li><a href=""><span class="socicon">j</span></a></li>
+                    <li><a href=""><span class="socicon">c</span></a></li>
+                </ul>
+                <p class="team-para">Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+            </li>
+            <li>
+                <li class="flex-item-4">
+                    <h2>JANE ELLA DOE</h2>
+                    <h3 class="team-title">UX specialist</h3>
+                    <img class="face-pic" src="./imgs/janedoe.png" alt="head picture of Jane Ella Doe">
+                    <ul class="social-media">
+                        <li><a href=""><span class="socicon">b</span></a></li>
+                        <li><a href=""><span class="socicon">a</span></a></li>
+                        <li><a href=""><span class="socicon">D</span></a></li>
+                        <li><a href=""><span class="socicon">j</span></a></li>
+                        <li><a href=""><span class="socicon">c</span></a></li>
+                    </ul>
+                    <p class="team-para">Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+                </li>
+                <li class="flex-item-4">
+                    <h2>JESSIE DOE</h2>
+                    <h3 class="team-title">Bussines and legacy</h3>
+                    <img class="face-pic" src="./imgs/jessiedoe.png" alt="head picture of Jessie Doe">
+                    <ul class="social-media">
+                        <li><a href=""><span class="socicon">b</span></a></li>
+                        <li><a href=""><span class="socicon">a</span></a></li>
+                        <li><a href=""><span class="socicon">D</span></a></li>
+                        <li><a href=""><span class="socicon">j</span></a></li>
+                        <li><a href=""><span class="socicon">c</span></a></li>
+                    </ul>
+                    <p class="team-para">Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+                </li>
+        </ul>
+        <h2 class="subheader">CLIENTS</h2>
+        <div class="client-wrapper">
+            <a href=""><img class="arrow" src="./imgs/l-arrow.png"></a>
```

Comment:
`img` for icons like this is so last week. More advanced method, please. `background-image` or pseudo-content for icons like this from here on out, please.